### PR TITLE
Remove backward compatibility codes for celery 3

### PR DESCRIPTION
**Pull request by Ladsgroup**

### CODE REVIEW

**Comment by Ladsgroup** [comment]
http://docs.celeryproject.org/en/v4.1.0/whatsnew-4.0.html#lowercase-setting-names

**Comment by Ladsgroup** [comment]
Don't merge. I need to find a fix for these three configs that are handled by puppet as well. This means we might end up having some down time or lots of root-level fixes with puppet disabled :/

**Comment by Ladsgroup** [comment]
This is done in another way \o/

File: config/celery-test-docker/00-main.yaml [hunk 1]
```diff
@@ -50,25 +50,25 @@ scoring_systems:
...
   docker_celery:
     class: ores.scoring_systems.CeleryQueue
     score_cache: docker_redis
-    BROKER_URL: redis://redis
-    BROKER_TRANSPORT_OPTIONS: {'socket_timeout': 15} # seconds
-    CELERY_RESULT_BACKEND: redis://redis
-    CELERY_ACCEPT_CONTENT: ['pickle']
-    CELERY_TASK_SERIALIZER: 'pickle'
-    CELERY_RESULT_SERIALIZER: 'pickle'
+    BROKER_URL: redis://localhost
```
Comment (halfak) [inline config/celery-test-docker/00-main.yaml]: This looks like a more substantial change.  WHat's going on here?

File: config/celery-test-docker/00-main.yaml [hunk 2]
```diff
@@ -50,25 +50,25 @@ scoring_systems:
...
+    broker_transport_options: {'socket_timeout': 15} # seconds
+    CELERY_RESULT_BACKEND: redis://localhost
```
Comment (halfak) [inline config/celery-test-docker/00-main.yaml]: CELERY_RESULT_BACKEND --> result_backend

File: config/celery-test-docker/00-main.yaml (hunk 1 above)
Comment (halfak) [reply config/celery-test-docker/00-main.yaml]: BROKER_URL --> broker_url

File: config/celery-test-docker/00-main.yaml [hunk 3]
```diff
@@ -50,25 +50,25 @@ scoring_systems:
...
-    BROKER_TRANSPORT_OPTIONS: {'socket_timeout': 15} # seconds
+    broker_transport_options: {'socket_timeout': 15} # seconds
     CELERY_RESULT_BACKEND: redis://localhost
-    CELERY_ACCEPT_CONTENT: ['pickle']
-    CELERY_TASK_SERIALIZER: 'pickle'
-    CELERY_RESULT_SERIALIZER: 'pickle'
+    accept_content: ['pickle']
+    task_serializer: 'pickle'
+    result_serializer: 'pickle'
     CELERYD_CONCURRENCY: 8
```
Comment (halfak) [inline config/celery-test-docker/00-main.yaml]: CELERYD_CONCURRENCY --> worker_concurrency

File: config/celery-test-docker/00-main.yaml [hunk 4]
```diff
@@ -50,25 +50,25 @@ scoring_systems:
     class: ores.scoring_systems.CeleryQueue
     score_cache: local_redis
     BROKER_URL: redis://localhost
-    BROKER_TRANSPORT_OPTIONS: {'socket_timeout': 15} # seconds
+    broker_transport_options: {'socket_timeout': 15} # seconds
     CELERY_RESULT_BACKEND: redis://localhost
```
Comment (halfak) [inline config/celery-test-docker/00-main.yaml]: CELERY_RESULT_BACKEND --> result_backend

File: config/celery-test-docker/00-main.yaml (hunk 2 above)
Comment (Ladsgroup) [reply config/celery-test-docker/00-main.yaml]: You can find the list in http://docs.celeryproject.org/en/v4.1.0/whatsnew-4.0.html#v400-upgrade-settings and http://docs.celeryproject.org/en/v4.1.0/userguide/configuration.html#new-lowercase-settings

**Pull request closed without merging**